lib/galaxy/datatypes/data.py

```diff
@@ -201,10 +202,10 @@ def display_peek(self, dataset ):
                 line = line.strip()
                 if not line:
                     continue
-                if isinstance(line, unicode):
+                if isinstance(line, text_type):
                     out.append( '<tr><td>%s</td></tr>' % escape( line ) )
                 else:
-                    out.append( '<tr><td>%s</td></tr>' % escape( unicode( line, 'utf-8' ) ) )
+                    out.append( '<tr><td>%s</td></tr>' % escape( text_type( line, 'utf-8' ) ) )
```

Comment:
I guess we would make the least amount of assumptions by using `text_type(var.encode('utf-8'))`.
This works in python 2 and 3.